src/NzbDrone.Core/Download/Clients/rTorrent/RTorrentSettings.cs

```diff
@@ -64,7 +64,7 @@ public RTorrentSettings()
         [FieldDefinition(10, Label = "Older Priority", Type = FieldType.Select, SelectOptions = typeof(RTorrentPriority), HelpText = "Priority to use when grabbing episodes that aired over 14 days ago")]
         public int OlderTvPriority { get; set; }
 
-        [FieldDefinition(11, Label = "Add Stopped", Type = FieldType.Checkbox, HelpText = "Enabling will prevent magnets from downloading before downloading")]
+        [FieldDefinition(11, Label = "Add Stopped", Type = FieldType.Checkbox, HelpText = "Enabling will add torrents and magnets to ruTorrent in a stopped state")]
```

Comment:
The old wording is clearly wrong, but it looks like it might cause issues with resolving magnets before the download starts, if that's the case we might want to be more specific about that.

Reply:
I have added some text specifying that it may break magnet links.